resources/styles/global/typography.less

```diff
@@ -63,6 +63,9 @@
     font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
     font-weight: 900;
   }
+  .scores-cells(){
+    font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;  
+  }
 }
```

Comment:
Pretty sure you're just keeping the existing font here, but shouldn't we be using Lato? I'm guessing probably not though